.github/workflows/deploy_testnet.yml

```diff
@@ -0,0 +1,20 @@
+name: testnet deploy
+
+on:
+  release:
+    types: [created]
+  workflow_dispatch:
```

Comment:
Manually triggering this probably won't work as `github.ref_name` won't be a useable git tag.